packages/model-viewer/src/test/features/animation-spec.ts

```diff
@@ -181,6 +182,32 @@ suite('ModelViewerElementBase with AnimationMixin', () => {
         });
       });
 
+      suite('with a specified index as animation-name', () => {
+        setup(async () => {
+          element.animationName = "1";
+          await timePasses();
+        });
+
+        test('plays the specified animation', () => {
+          expect(animationIsPlaying(element, element.availableAnimations[1]))
+              .to.be.true;
+        });
+      });
+
+      suite('a model with duplicate animation names', () => {
+        setup(async () => {
+          element.src = ANIMATED_GLB_DUPLICATE_ANIMATION_NAMES_PATH;
+          await waitForEvent(element, 'load');
+          element.animationName = "1";
+          await timePasses();
+        });
+
+        test('plays the specified animation', () => {
+          expect(animationIsPlaying(element, element.availableAnimations[1]))
```

Comment:
This will always succeed for a model with duplicate clip names. Correct would be to check that "1" plays animations[1].

Reply:
In fact all the internals *are* available to our tests, since private things are just symbols; we often import a few symbols to our tests for this purpose. 

I'm impressed you're managing this without being able to run the tests locally; any idea what the trouble is? I'd like to ensure our project is runnable from WSL2. Granted our Windows 10 setup instructions are a bit light, but if something is missing we should definitely add it.

Reply:
I started a readme with some of the things that I tried, and some of them that worked - think I have a working test setup now and will attempt the above again.

Is there a good place to PR those notes to? Would be great to link to it from the main Readme WSL section.
(I could also just send you a link and you add it to the wiki?)

Reply:
You're welcome to add them directly to the WSL section of the README if it's not crazy long. But yeah, I can add it to the wiki otherwise. I just feel like the wiki overall is not terrible up to date...